lib/Extension/LanguageServer/Listener/SelfDestructListener.php

```diff
@@ -33,7 +33,7 @@ public function handleShutdown(WillShutdown $willShutdown): void
         asyncCall(function () {
             yield delay($this->selfDestructTimeout);
             throw new ExitSession(sprintf(
-                'Waited "%s" seconds after shutdown request for exit notification but did not get one so I\'m self destructing.',
+                'Waited "%s" milliseconds after shutdown request for exit notification but did not get one so I\'m self destructing.',
```

Comment:
Good catch. If you want you could also update the documentation to show the units. (LanguageServerExtension.php:151)

Reply:
I'll update the CL and fix this, thanks @przepompownia !

Reply:
(and @mamazu for the ground work!)

Reply:
Thanks to both of you too!